static/js/components/MITLogoLink.js

```diff
@@ -1,16 +1,11 @@
 // @flow
-/* global SETTINGS: false */
 import React from "react"
 
+import { MIT_LOGO_URL } from "../lib/url"
+
 const MITLogoLink = () => (
   <a href="http://www.mit.edu" className="mitlogo">
-    <img
-      src={`/static/images/${
-        SETTINGS.use_new_branding
-          ? "MIT_circle.svg"
-          : "mit-logo-transparent3.svg"
-      }`}
-    />
+    <img src={MIT_LOGO_URL} />
```

Comment:
While we're here, could you add an `alt`?

Reply:
sure, do you think `"mit logo"` covers it?